config/default.yml

```diff
@@ -479,8 +479,9 @@ RSpec/ImplicitSubject:
     - single_line_only
     - single_statement_only
     - disallow
+    - always
```

Comment:
The rest of the styles either disallow the usage of implicit subject or allow it in certain situations. This doesn't just allows the usage, but requires it. Note that single_line_only doesn't require the use of `is_expected` in single line examples, it merely allows it. So if we are to add this option, I'd go with something like `require_implicit`. Or it may be extracted in separate option (`require_implicit`) and then `always` will be one that either always allow (require_implicit being false) or always requires (if set to true)

Reply:
`require_implicit` seems to be more preferable. Changed it that way 👍